src/foam/nanos/pii/PIIReportTicket.js

```diff
@@ -0,0 +1,116 @@
+/**
+ * @license
+ * Copyright 2024 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.pii',
+  name: 'PIIReportTicket',
+  extends: 'foam.nanos.ticket.Ticket',
+
+  documentation: 'Generate a PII Report for a user',
+
+  javaImports: [
+    'foam.nanos.fs.File'
+  ],
+
+  properties: [
+    {
+      name: 'title',
+      value: 'PII Report Request'
+    },
+    {
+      name: 'status',
+      order: 5,
+      createVisibility: 'HIDDEN'
+    },
+    {
+      name: 'statusChoices',
+      hidden: true,
+      factory: function() {
+        var s = [];
+        if ( 'CLOSED' == this.status ) {
+          s.push(['CLOSED', 'CLOSED']);
+          s.push(['OPEN', 'OPEN']);
+        } else {
+          s.push(this.status);
+          s.push(['CLOSED', 'CLOSED']);
+        }
+        return s;
+      }
+    },
+    {
+      name: 'comment',
+      order: 6
+    },
+    {
+      name: 'createdFor',
+      gridColumns: '12'
+    },
+    {
+      name: 'state',
+      hidden: true,
+      transient: true
+    },
+    {
+      name: 'assignedToGroup',
+      hidden: true
+    },
+    {
+      name: 'externalComment',
+      hidden: true
+    },
+    {
+      class: 'foam.nanos.fs.FileArray',
+      name: 'documents',
+      createVisibility: 'HIDDEN',
+      readVisibility: 'RO',
+      updateVisibility: 'RO',
+      section: 'infoSection',
+      order: 15,
+      gridColumns: 12,
+      tableCellFormatter: function(documents) {
+        if ( ! (Array.isArray(documents) && documents.length > 0) ) return;
+        var actions = documents.map((file) => {
+          return foam.core.Action.create({
+            label: file.filename,
+            code: function() {
+              window.open(file.address, '_blank');
```

Comment:
should probably do function(x) { x.window.open...

Reply:
replaced